src/main/java/codeu/model/data/User.java

```diff
@@ -59,8 +59,13 @@ public User(UUID id, String name, String passwordHash, Instant creation, long me
    * @param creation     the creation time of this User
    * @param isAdmin      if the user is an admin or not
    */
-  public User(UUID id, String name, String passwordHash, Instant creation, boolean isAdmin) {
-    this(id, name, passwordHash, creation, 0, isAdmin, "");
+  public User(UUID id, String name, String passwordHash, Instant creation) {
+    this(id, name, passwordHash, creation, 0, false, "");
+  }
+
+  private Boolean checkIfIsAdmin(String name) {
+    return name.equals("Ricardo") || name.equals("Manjil") 
```

Comment:
Can we make this a static list in this class? And maybe we can name it something like "defaultAdminList" because this list doesnt contain all admins, and we dont update it to contain all admins.

Reply:
Ok, but I will add the admin names hard code, I don't want to add the Guava library for one line of code, even if it really is the best.

Reply:
done